content/product/product_management/index.md

```diff
@@ -20,15 +20,20 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
 
 In addition to this shared content, each team has a set of sources of truth (that do not overlap) that they should update regularly:
 
-#### Per-area strategy pages
+#### OKR and roadmap tracking
 
-We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) to communicate where each of our product teams is headed, for the areas of the product that they work on. Newly created teams should begin documenting the scope of their team as well as their mission, vision, principles, and more by creating one of these pages as part of the process of forming. Having a page like this will help you ensure your own framework for prioritization is clear, and can also be a resource for those outside your team who want to understand why you are working on what you are.
+We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each month you should work with your engineering manager to update this content, which is the source of truth for goal and roadmap tracking.
 
-These are updated monthly by each team, but wouldn't normally change too much each month. They are updated via a pull request, that anyone on the team can contribute to, and that can be shared with stakeholders as a summary of the change.
+After updating this content, you should ensure it is reflected in the manually created slide decks which rely on this information:
+
+- The [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming and recently launched important customer-facing features from the OKR and roadmap tracker.
```

Comment:
Question, if we have the content available in an internally available source of truth now, could the task of copying it to the deck for marketing purposes be done by the marketing team instead of the product managers?

Reply:
I think there is value in the product team making sure content is up to date and correct, but it does make sense for the PMM team to own.

Reply:
We can leave as is for now then @christinaforney and see if we want to adjust in the future if it makes sense.

```suggestion
- The [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming and recently launched important customer-facing features from the OKR and roadmap tracker. In the future we may transition updating this to PMM, but for now its important product managers are involved in ensuring it is up to date and correct.
```